docs/source/cli-reference/output-filters-and-control.rst

```diff
@@ -0,0 +1,101 @@
+.. _cli_output_control:
+
+Controlling Scancode Output and Filters
+=======================================
+
+.. include::  /rst_snippets/output_control_options.rst
+
+----
+
+.. include::  /rst_snippets/note_snippets/synopsis_install_quickstart.rst
+
+----
+
+``--strip-root`` Vs. ``--full-root``
+------------------------------------
+
+    For a default scan of the "samples" folder, this a comparison between the default,
+    ``strip-root`` and ``full-root`` options.
+
+    An example Scan
+
+    ::
+
+        scancode -cplieu --json-pp output.json samples --full-root
+
+    These two changes only the "path" attribute of the file information. For this comparison we
+    compare the "path" attributes of the file ``LICENSE`` inside ``JGroups`` directory.
+
+    The default path::
+
+        "path": "samples/JGroups/LICENSE",
+
+    For the ``--full-root`` option, the path relative to the Root of your local filesystem.
+
+    ::
+
+        "path": "/home/ayansm/Desktop/GSoD/scancode-toolkit-versions/scancode-toolkit-2.2.1/samples/JGroups/LICENSE"
+
+
+    For the ``--strip-root`` option, the root directory (here ``samples``) is removed from path :
+
+    ::
+
+        "path": "JGroups/LICENSE"
+
+    .. include::  /rst_snippets/note_snippets/control_strip_full_root.rst
+
+----
+
+``--ignore-author <pattern>`` Option
```

Comment:
We should also document the more popular `--ignore` option here as well.

Reply:
So that's grouped under[ `Pre-Scan` Options](https://aboutcode.readthedocs.io/en/latest/scancode-toolkit/cli-reference/list-options.html#all-pre-scan-options). Again, here I'm following the groups of options to be consistent with the help text.

Reply:
Yes, I suppose following the various groups is fine. I had forgotten that this is how they are structured internally in the code itself.